src/main/java/software/bernie/techarium/trait/block/BlockBehaviours.java

```diff
@@ -0,0 +1,35 @@
+package software.bernie.techarium.trait.block;
+
+import software.bernie.techarium.registry.LangRegistry;
+import software.bernie.techarium.tile.arboretum.ArboretumTile;
+import software.bernie.techarium.tile.botarium.BotariumTile;
+import software.bernie.techarium.tile.exchangestation.ExchangeStationTile;
+
+public class BlockBehaviours {
```

Comment:
Make them final and CAPS LOCK all of them

Reply:
Also add an automatable behaviour/Trait for the Assembler, so it doesn't expose item capability, if the trait/behaviour is missing

Reply:
I'm not sure what you mean by that

Reply:
Nvmd, me dumb at 7am